controls/V-92985.rb

```diff
@@ -39,25 +37,35 @@
   impact 0.5
   tag 'severity': nil
   tag 'gtitle': 'SRG-OS-000004-GPOS-00004'
-  tag 'satisfies': ["SRG-OS-000004-GPOS-00004", "SRG-OS-000239-GPOS-00089",
-"SRG-OS-000240-GPOS-00090", "SRG-OS-000241-GPOS-00091",
-"SRG-OS-000303-GPOS-00120", "SRG-OS-000476-GPOS-00221"]
+  tag 'satisfies': %w(SRG-OS-000004-GPOS-00004 SRG-OS-000239-GPOS-00089
+SRG-OS-000240-GPOS-00090 SRG-OS-000241-GPOS-00091
+SRG-OS-000303-GPOS-00120 SRG-OS-000476-GPOS-00221)
   tag 'gid': 'V-92985'
   tag 'rid': 'SV-103073r1_rule'
   tag 'stig_id': 'WN19-DC-000230'
   tag 'fix_id': 'F-99231r1_fix'
-  tag 'cci': ["CCI-000018", "CCI-000172", "CCI-001403", "CCI-001404",
-"CCI-001405", "CCI-002130"]
-  tag 'nist': ["AC-2 (4)", "AU-12 c", "AC-2 (4)", "AC-2 (4)", "AC-2 (4)", "AC-2
-(4)", "Rev_4"]
-
-  describe.one do
-    describe audit_policy do
-      its('Computer Account Management') { should eq 'Success' }
+  tag 'cci': %w(CCI-000018 CCI-000172 CCI-001403 CCI-001404
+CCI-001405 CCI-002130)
+  tag 'nist': ['AC-2 (4)', 'AU-12 c', 'AC-2 (4)', 'AC-2 (4)', 'AC-2 (4)', "AC-2
+(4)", 'Rev_4']
+
+  domain_role = command('wmic computersystem get domainrole | Findstr /v DomainRole').stdout.strip
+  case domain_role
+  when '4', '5'
+    impact 0.5
+    describe.one do
+      describe audit_policy do
+        its('Computer Account Management') { should eq 'Success' }
+      end
+      describe audit_policy do
+        its('Computer Account Management') { should eq 'Success and Failure' }
+      end
     end
-    describe audit_policy do
-      its('Computer Account Management') { should eq 'Success and Failure' }
+  when '2', '3'
+    impact 0.0
+    desc 'This system is exempt from this control'
+    describe 'This system is exempt from this control' do
+      skip 'This system is exempt from this control'
     end
   end
```

Comment:
Great catch! The case statements you've added are just fine but to standardize it with other controls, I'd recommend using:

```
if domain_role == '4' || domain_role == '5'
  describe.one do
    describe audit_policy do
      its('Computer Account Management') { should eq 'Success' }
    end
    describe audit_policy do
      its('Computer Account Management') { should eq 'Success and Failure' }
    end
  end
else
  impact 0.0
  describe 'This system is not a domain controller, therefore this control is not applicable' do
    skip 'This system is not a domain controller, therefore this control is not applicable'
  end
end
```
Note: Since the impact is already set to 0.5 above the tags, we don't need to set it again for the domain controller condition

Reply:
I see that you have already updated this @aagiardd so I'm resolving this comment. Thanks!